openedx/core/djangoapps/catalog/utils.py

```diff
@@ -543,3 +543,56 @@ def get_course_run_details(course_run_key, fields):
         course_run_details = get_edx_api_data(catalog_integration, 'course_runs', api, resource_id=course_run_key,
                                               cache_key=cache_key, many=False, traverse_pagination=False, fields=fields)
     return course_run_details
+
+
+def course_run_keys(parent_program):
```

Comment:
#NIT how about `course_runs_keys_of_program` as the function name?

Reply:
yeah, good call.